grails-app/views/putAway/_print.gsp

```diff
@@ -124,9 +142,15 @@
                         <wordwrap:td>${putawayItem["inventoryItem.expirationDate"]}</wordwrap:td>
                         <wordwrap:td>${putawayItem?.quantity}</wordwrap:td>
                         <wordwrap:td>${putawayItem?.quantity}</wordwrap:td>
-                        <wordwrap:td>${(putawayItem["preferredBin.zoneName"] ? putawayItem["preferredBin.zoneName"] + ": " : '') + putawayItem["preferredBin.name"]}</wordwrap:td>
+                        <wordwrap:td>
+                            ${(putawayItem["preferredBin.zoneName"] ? putawayItem["preferredBin.zoneName"] + ": " : '') +
+                                    (putawayItem["preferredBin.name"] ?: g.message(code: "default.null.label", default: "null"))}
```

Comment:
Why this is null instead of just empty?

Reply:
In the future, please don't resolve the discussion without a response.

Reply:
@jmiranda we answered that on our internal standup and also it was answered by Kelsey on the develop channel after Artur's question. I haven't expected anyone (including you) outside of the discussion we had about this, to jump into the PR, hence I didn't see a point to comment on that here.
I apologize for that.